### DISCUSSION TITLE
Pingvin Share fails to update (Error in line 71)

### DISCUSSION THREAD

**Discussion by Node815** [tteck/Proxmox#3407] [Help Needed]
I need to update Pinvgin Share and am getting the following error while running the `update` command inside the LXC:

 ✓ Stopped Pingvin Share
 \ Updating Pingvin Share   
[ERROR] in line 71: exit code 0: while executing command npm run build &> /dev/null

root@pingvin:~# 

How is this updated then?

**Comment by Mellowlynx** [comment]
This should fix it: https://github.com/tteck/Proxmox/discussions/3300

**Reply by Node815** [reply]
Sadly no:
 ✓ Stopped Pingvin Share
 - Updating Pingvin Share   
[ERROR] in line 71: exit code 0: while executing command npm run build &> /dev/null

 
I ran the command in #3300 as well.  

From Proxmox command line:

root@pve:~# lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 12 (bookworm)
Release:        12
Codename:       bookworm

I don't see the point release listed in the above.

I don't recall updating to Debian 12.6, unless it was done in the background by Proxmox.  For now, I reinstalled it using the Pingvin shares script.  I use SSO to sign into it and wanted to get the new version to disable the login page which it dos now.  But, the reconfig is what I was hoping to avoid.

**Comment by itsdoublearon** [comment]
I'm getting the same error when trying to update as well.

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/3407#discussioncomment-10103604 and/or increase resources for the build.

**Reply by itsdoublearon** [reply]
Doubled all resources, rebooted, still the error ([ERROR] in line 68: exit code 0: while executing command) ran the npm run build, still nothing, another reboot, still an error.

**Reply by Node815** [reply]
It seems they are developing a bit faster than before so I checked for an update and found: 
https://github.com/stonith404/pingvin-share/releases/tag/v0.29.0
As a workaround to #3407 I just redeployed the install to a new LXC and removed the old one.  This put the 2.7.0 version on which I use now. 

I encountered a new error which I'm not sure if this will this issues stills applies or not or if it's new one based on the line number:
```    ____  _                   _          _____ __                  
   / __ \(_)___  ____ __   __(_)___     / ___// /_  ____ _________
  / /_/ / / __ \/ __ `/ | / / / __ \    \__ \/ __ \/ __ `/ ___/ _ \
 / ____/ / / / / /_/ /| |/ / / / / /   ___/ / / / / /_/ / /  /  __/
/_/   /_/_/ /_/\__, / |___/_/_/ /_/   /____/_/ /_/\__,_/_/   \___/
              /____/
 ✓ Stopped Pingvin Share
 \remote: Enumerating objects: 364, done.
remote: Counting objects: 100% (363/363), done.
remote: Compressing objects: 100% (190/190), done.
remote: Total 364 (delta 214), reused 290 (delta 169), pack-reused 1
Receiving objects: 100% (364/364), 526.77 KiB | 20.26 MiB/s, done.
Resolving deltas: 100% (214/214), completed with 17 local objects.
 /From https://github.com/stonith404/pingvin-share
 + 4c0aabc...ba7a5d2 i18n_crowdin -> origin/i18n_crowdin  (forced update)
   e5e9d85..f1f514d  main         -> origin/main
 * [new tag]         v0.28.0      -> v0.28.0
 * [new tag]         v0.29.0      -> v0.29.0

[ERROR] in line 65: exit code 0: while executing command git checkout $(git describe --tags `git rev-list --tags --max-count=1`) &> /dev/null

root@pingvin:~# 
```

I assume this resources are sufficient?
![image](https://github.com/user-attachments/assets/4895040f-17de-40bb-b2ac-9271720b1e59)
